sda-commons-server-jackson/src/test/java/org/sdase/commons/server/jackson/JacksonConfigurationInvalidTypeBundleIT.java

```diff
@@ -0,0 +1,63 @@
+package org.sdase.commons.server.jackson;
+
+import static io.dropwizard.testing.ConfigOverride.randomPorts;
+
+import io.dropwizard.Configuration;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.groups.Tuple;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.sdase.commons.server.jackson.test.*;
+import org.sdase.commons.shared.api.error.ApiError;
+
+class JacksonConfigurationInvalidTypeBundleIT {
+
+  @RegisterExtension
+  public static final DropwizardAppExtension<Configuration> DW =
+      new DropwizardAppExtension<>(
+          JacksonConfigurationInvalidTypeTestApp.class, null, randomPorts());
+
+  @Test
+  void shouldPostResourceWithInheritance() {
+    ResourceWithInheritance resource = new ResourceWithInheritanceSubType();
```

Comment:
I would prefer to use an unrelated Json in the test, e.g. a Json created from `Map<String, Object>` or a bare Json String, so that the test data is not influenced by the tested resource.

Reply:
Done